templates/settings.yaml.erb

```diff
@@ -12,3 +12,11 @@
 :oauth_map_users: true
 :oauth_consumer_key: <%= scope['::katello_devel::oauth_key'] %>
 :oauth_consumer_secret: <%= scope['::katello_devel::oauth_secret'] %>
+
+# SSL settings
+:ssl_ca_file: <%= scope['::katello_devel::foreman_certs::ssl_ca_cert'] %>
+:ssl_certificate: <%= scope['::katello_devel::foreman_certs::client_cert'] %>
+:ssl_priv_key: <%= scope['::katello_devel::foreman_certs::client_key'] %>
+
+# Webpack settings
+:webpack_dev_server: false
```

Comment:
nit: could you also add `:webpack_dev_server_https: false` pls?

Reply:
Done